packages/kit/src/runtime/client/client.js

```diff
@@ -459,6 +459,7 @@ export function create_client({ target, base, trailing_slash }) {
 				routeId: route && route.id,
 				status,
 				url,
+				form,
```

Comment:
The `page_changed` check above probably needs to be enhanced by appending `|| form !== undefined`